policyengine_canada/parameters/gov/provinces/yt/benefits/rebates/ygcpri/amount/child.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yukon provides this base amount for each child under the Yukon Government carbon price rebate (YGCPRI).
+values:
+  2023-01-01: 320
```

Comment:
Now 372. Can you find historical parameter values so we can assign the correct date?